app/assets/stylesheets/application.scss

```diff
@@ -340,7 +348,7 @@ table {
 .spinner-placeholder {
   width: 16px;
   height: 16px;
-  background: image-url('spinner.gif');
+  background: url('spinner.gif');
```

Comment:
no, i mean the file is actually deleted in this pr. (and you are right about the url vs image-url).